config/samples/0.9.1/cluster_v1_demo.yaml

```diff
@@ -15,6 +15,16 @@ spec:
   primaryMasters:
     instanceCount: 3
     cellTag: 1
+    affinity:
```

Comment:
without it, this error will appear:

```
The Ytsaurus "ytdemo" is invalid: spec.primaryMasters.affinity.podAntiAffinity: Required value: Masters should be placed on different nodes
```

i added fix to 0.9.1 and to file without version (is it latest version?), don't know if it's applicable to previous version